### PR TITLE
Add web app manifest and launcher icons

### PR DESCRIPTION
**Pull request by kernel-killer**
This allows the web page to look as a native app and hides the browser's UI, which together with music controls at the bottom may occupy quiet a bit of screen space, especially in phones with smaller screens. Works in Android and iOS (using 3rd party browsers - Chrome, Firefox..., the support may be extended to more browsers using [pwacompat library](https://github.com/GoogleChrome/pwacompat)).

### CODE REVIEW

**Comment by kernel-killer** [comment]
The icons are low-resolution images (48x48px, 96x96px, 192x192px), down-scaled using "Scale" feature in GIMP, Interpolation mode set to "Sinc (Lanczos 3)" - provides the best output.

**Comment by jb55** [comment]
hmm scaling a rasterized image wouldn't be the right approach here. I suggest using our vector assets here: https://assets.monstercat.com/essentials/logos/monstercat_logo_cat.svg

this is very cool! with that change we should be able to include this.

**Comment by tmathews** [comment]
Needs update for new build process I believe. Will merge once sorted.

**Comment by jb55** [comment]
@tmathews should wait for rasterized vector assets before we merge

File: scripts/build [hunk 1]
```diff
@@ -33,6 +33,9 @@ ln -s $SRC/js          $BIN/js
 rm -f $BIN/favicon.ico
 ln -s $SRC/favicon.ico $BIN/favicon.ico
 
+rm -f $PWD/bin/manifest.json
```
Comment (jb55) [inline scripts/build]: these should be $BIN instead of $PWD/bin and $SRC instead of $PWD/src

**Comment by jb55** [comment]
Thanks squashed and merged!

**Pull request closed without merging**